roles/edpm_ovn/tasks/install.yml

```diff
@@ -15,6 +15,7 @@
 # under the License.
 
 - name: Create persistent directories
+  become: true
```

Comment:
ah this is where the incorrect permission are coming form this should be setting the owner to ansibe_user or ansible_user_id preferably with the double defaulting so that both will be checked

Reply:
Do we have another preferred owner for these? Otherwise there is no need for double defaulting. Just the `ansible_user` and `ansible_user_id` as a fallback.